contracts/jobs/JobsManager.sol

```diff
@@ -104,49 +104,14 @@ contract JobsManager is ManagerProxyTarget, IVerifiable, IJobsManager {
 
     function JobsManager(address _controller) public Manager(_controller) {}
 
-    /*
-     * @dev Batch set protocol parameters. Only callable by the controller owner
-     * @param _verificationRate % of segments to be verified
-     * @param _verificationPeriod Number of blocks to complete verification of claimed work
-     * @param _slashingPeriod Number of blocks after the verification period to submit slashing proofs
-     * @param _failedVerificationSlashAmount % of stake slashed for failed verification
-     * @param _missedVerificationSlashAmount % of stake slashed for missed verification
-     * @param _doubleClaimSegmentSlashAmount % of stake slashed of double claiming a segment
-     * @param _finderFee % of slashed amount awawded to finder
-     */
-    function setParameters(
```

Comment:
What's the reason for removing this? We have individual setters, so we can reduce contract size and redundant functionality that needs to be maintained?

Reply:
Yeah primarily to reduce contract size and redundant functionality